bin/collator/src/local/service.rs

```diff
@@ -310,24 +315,47 @@ where
 
     // Frontier offchain DB task. Essential.
     // Maps emulated ethereum data to substrate native data.
-    task_manager.spawn_essential_handle().spawn(
-        "frontier-mapping-sync-worker",
-        Some("frontier"),
-        fc_mapping_sync::kv::MappingSyncWorker::new(
-            client.import_notification_stream(),
-            Duration::new(6, 0),
-            client.clone(),
-            backend.clone(),
-            storage_override.clone(),
-            frontier_backend.clone(),
-            3,
-            0,
-            fc_mapping_sync::SyncStrategy::Parachain,
-            sync_service.clone(),
-            pubsub_notification_sinks.clone(),
-        )
-        .for_each(|()| futures::future::ready(())),
-    );
+    match frontier_backend.as_ref() {
```

Comment:
Suggestion: Maybe we can make an helper for consistency? (`spawn_mapping_sync_worker`)

Reply:
Extract it as a helper function will takes 7 parameters. The clean code way would be to create config structs to pass to helpers functions. But not needed IMO. What do you think ?

Reply:
Too much parameters, we can keep it as it is